dearpygui/dearpygui.py

```diff
@@ -9501,17 +9692,41 @@ def unstage(item : Union[int, str], **kwargs) -> None:
 mvKey_F22=internal_dpg.mvKey_F22
 mvKey_F23=internal_dpg.mvKey_F23
 mvKey_F24=internal_dpg.mvKey_F24
-mvKey_F25=internal_dpg.mvKey_F25
 mvKey_NumLock=internal_dpg.mvKey_NumLock
 mvKey_ScrollLock=internal_dpg.mvKey_ScrollLock
-mvKey_LShift=internal_dpg.mvKey_LShift
-mvKey_RShift=internal_dpg.mvKey_RShift
-mvKey_LControl=internal_dpg.mvKey_LControl
-mvKey_RControl=internal_dpg.mvKey_RControl
-mvKey_LMenu=internal_dpg.mvKey_LMenu
-mvKey_RMenu=internal_dpg.mvKey_RMenu
+mvKey_Period=internal_dpg.mvKey_Period
+mvKey_Slash=internal_dpg.mvKey_Slash
+mvKey_Backslash=internal_dpg.mvKey_Backslash
+mvKey_Open_Brace=internal_dpg.mvKey_Open_Brace
+mvKey_Close_Brace=internal_dpg.mvKey_Close_Brace
 mvKey_Browser_Back=internal_dpg.mvKey_Browser_Back
 mvKey_Browser_Forward=internal_dpg.mvKey_Browser_Forward
+mvKey_Comma=internal_dpg.mvKey_Comma
+mvKey_Minus=internal_dpg.mvKey_Minus
+mvKey_Menu=internal_dpg.mvKey_Menu
+mvKey_ModSuper=internal_dpg.mvKey_ModSuper
+mvKey_ModShift=internal_dpg.mvKey_ModShift
+mvKey_ModAlt=internal_dpg.mvKey_ModAlt
+mvKey_ModCtrl=internal_dpg.mvKey_ModCtrl
+mvKey_Prior=internal_dpg.mvKey_Prior
+mvKey_Next=internal_dpg.mvKey_Next
+mvKey_Select=internal_dpg.mvKey_Select
+mvKey_Execute=internal_dpg.mvKey_Execute
+mvKey_LWin=internal_dpg.mvKey_LWin
+mvKey_RWin=internal_dpg.mvKey_RWin
+mvKey_Apps=internal_dpg.mvKey_Apps
+mvKey_Sleep=internal_dpg.mvKey_Sleep
+mvKey_Clear=internal_dpg.mvKey_Clear
```

Comment:
Thank you, I didn't notice this. With the commit [760eedf](https://github.com/hoffstadt/DearPyGui/pull/2275/commits/760eedf2b27fc116d3fe38a2f620f162c79c3568) I should have fixed it.
Also I set all the "old" buttons to `ImGuiKey_None` when `IMGUI_DISABLE_OBSOLETE_FUNCTIONS` is enabled. You tell me if it's good choice or not.

Reply:
Do we really need that `IMGUI_DISABLE_OBSOLETE_FUNCTIONS` branch? I doubt that anyone is going to build DPG with that macro, but since it's harmless I'm okay with keeping it.

Reply:
Okay, I wanted to keep it for people who wanted to play around it, but if you say it's useless no problem